### ISSUE TITLE
Anima sum per character and total

### ISSUE THREAD

**Issue by andybergon** [AzerPUG/Renown-Checker#3]
In the AltFrame it would be nice to also show the total anima per character and total per account.

![image](https://user-images.githubusercontent.com/5220825/155899634-d5b10494-67f3-4dff-9231-9b85d2c66857.png)

**Comment by Arco-Dielhof** [comment]
Not sure what the use would be, since anima within one covenant is no longer able to change to another, right? Or am I missing something? However, I can look into making that an option for those that want it!

Thank you for the suggestion! :D

**Comment by andybergon** [comment]
> anima within one covenant is no longer able to change to another

You can still buy the 1k anima object with one covenant and use it in another. 
So if for example you have to do un an anima upgrade that costs 15k you could quickly check that just swapping covenants and buying anima transfer items would be enough. 

In theory what you are interested in it's not the simple sum as you can only buy and send 1k increments so there could be something like `Transferrable Anima: N k` where `N = sum(anima_cov1 // 1000 + anima_cov2 // 1000 + ...)` (`//` being the division without reminder).

I would like to help but I never developed WoW addons and would need to do some research first.

**Comment by andybergon** [comment]
Side feature on this: also track `Traveler's Anima Cache` in bag and count them in the anima total. 
This is a bit up to debated as it moves this addon to be more an anima tracker than just renown tracker but I think a lot of people (like me) started to use this to track anima more than just the renown level.

**Comment by Arco-Dielhof** [comment]
I accepted a while ago, that although it is called "Renown Checker", that it is a bit more than that by now. Also because of feature requests (the anima per covenant was also a request from a use :P ). That said, I do not actually mind that at all. I am glad I can provide people with something that they need/want.

Since it now actually make sense, thanks your explanation, I will put this as well on my March Goals, and hope to get it to you, somewhat soon ^^

The Traveler's Anima Cache should not be to hard to add (I got another AddOn that already tracks openable items and give an easy way to open them, although I have not updated that to include everything yet.)
I'll consider this as well, after I fixed the other two of your requests :)

Once again, thanx for the suggestions!
